tests/mdev_configuration_allocation_test.go

```diff
@@ -118,14 +118,16 @@ var _ = Describe("[Serial][sig-compute]MediatedDevices", func() {
 		var desiredMdevTypeName string = "nvidia-222"
 		var expectedInstancesNum int = 16
 		var config v1.KubeVirtConfiguration
+		var originalConfig v1.KubeVirtConfiguration
 		var mdevTestLabel = "mdevTestLabel1"
 
 		BeforeEach(func() {
 			kv := util.GetCurrentKv(virtClient)
 
 			By("Creating a configuration for mediated devices")
 			config = kv.Spec.Configuration
-			config.DeveloperConfiguration.FeatureGates = []string{virtconfig.GPUGate}
```

Comment:
:+1: 
We should converge to the usage of EnableFeatureGate but not here. Great catch!

Reply:
:+1: Thanks